diff -uNr qemu-6.1.0/meson.build qemu-6.1.0.mod/meson.build
--- qemu-6.1.0/meson.build	2021-08-25 19:52:27.673921612 +0300
+++ qemu-6.1.0.mod/meson.build	2021-08-25 20:09:20.496591469 +0300
@@ -310,7 +310,7 @@
 add_project_arguments(config_host['GLIB_CFLAGS'].split(),
                       native: false, language: ['c', 'cpp', 'objc'])
 glib = declare_dependency(compile_args: config_host['GLIB_CFLAGS'].split(),
-                          link_args: config_host['GLIB_LIBS'].split())
+                          link_args: config_host['GLIB_LIBS'].split() + ['-liconv'])
 # override glib dep with the configure results (for subprojects)
 meson.override_dependency('glib-2.0', glib)
 
